Examples/test-suite/common.mk

```diff
@@ -700,6 +705,7 @@ all: $(NOT_BROKEN_TEST_CASES) $(BROKEN_TEST_CASES)
 broken: $(BROKEN_TEST_CASES)
 
 check: $(NOT_BROKEN_TEST_CASES)
+	@echo $(shell echo $^|wc -w) tests passed
```

Comment:
You can easily avoid the overhead of running external processes here (and avoid relying on `wc` being available):

```
@echo $(words $^) tests passed
```

Personally what I've done for this case is to run `make check-foo-test-suite > check-foo.new 2>&1` each time, and keep the output for the currently committed version in `check-foo.current`.  Then to see what difference a change I'm working on makes, I can `diff check-foo.current check-foo.new`.  The advantage of this approach is you get to see when tests move from failing to passing due to a change, while the above change seems to just prevent a blacklist of tests from running at all.

I guess the advantage of this is that automated builds can run the testsuite and understand if the outcome is a regression.